packages/css/src/transformCss.test.ts

```diff
@@ -1067,6 +1067,359 @@ describe('transformCss', () => {
     `);
   });
 
+  it('should handle @layer declarations', () => {
+    expect(
+      transformCss({
+        composedClassLists: [],
+        localClassNames: ['testClass'],
+        cssObjs: [
+          {
+            type: 'layer',
+            name: 'reset',
+          },
+          {
+            type: 'layer',
+            name: 'foo.bar.baz',
+          },
+        ],
+      }).join('\n'),
+    ).toMatchInlineSnapshot(`
+      "@layer reset;
+      @layer foo.bar.baz;"
+    `);
+  });
+
+  it('should not write layer declarations if not required', () => {
```

Comment:
This test is no longer valid. Snapshot is correct though.

Reply:
Done here https://github.com/vanilla-extract-css/vanilla-extract/pull/1050/files#diff-c2f03bedebd25a849a4e93e555d6c30b22df595ed84055d6083c5beaa5e8c986R1098